service/controller/app/resource/chart/resource.go

```diff
@@ -124,8 +125,13 @@ func copyChart(current *v1alpha1.Chart) *v1alpha1.Chart {
 // copyAnnotations copies annotations from the current to desired chart,
 // only if the key has a chart-operator.giantswarm.io prefix.
 func copyAnnotations(current, desired *v1alpha1.Chart) {
+	webhookAnnotation := fmt.Sprintf("%s/%s", annotation.ChartOperatorPrefix, annotation.WebhookURL)
+
 	for k, currentValue := range current.Annotations {
-		if !strings.HasPrefix(k, annotation.ChartOperatorPrefix) {
+		if k == webhookAnnotation {
+			// Remove webhook annotation that is no longer used.
+			continue
```

Comment:
Fixes a bug found in testing. We copy chart-operator annotations so we need to explicitly skip coping the webhook annotation to remove it.